### DISCUSSION TITLE
.

### DISCUSSION THREAD

**Discussion by RagenP** [Jip-Hop/jailmaker#249] [Q&A]
no longer relevant

**Comment by Lockszmith-GH** [comment]
There is very little visibility on how you setup your system, so this feels like an [XYproblem](https://xyproblem.info/),

To answer your questions of 'what' and 'why', much more technical detail should be presented.

Please reviewe #135. It has helpful information-collection instructions/script.
If you send as much detail as you can about your setup, the community will have a better chance of helping you out.
